test/metabase/task/search_index_test.clj

```diff
@@ -0,0 +1,24 @@
+(ns metabase.task.search-index-test
+  (:require
+   [clojure.test :refer :all]
+   [metabase.search.appdb.index :as search.index]
+   [metabase.search.test-util :as search.tu]
+   [metabase.task.search-index :as task]
+   [toucan2.core :as t2]))
+
+(defn- index-size []
+  (t2/count search.index/*active-table*))
+
+;; TODO this is coupled to appdb engines at the moment
```

Comment:
Not quite sure how to test this. We will test the individual indices, maybe this should just check that the appropriate engine methods get called? Kinda trivial.

Anyway, I just want to be sure the code is exercised for now.

Reply:
I guess a good test for this would be to delete an entity, and show that reindexing clears it from the index.